Markdown/Markdown.sublime-syntax

```diff
@@ -924,6 +924,7 @@ contexts:
     - match: |-
          (?x)
           {{fenced_code_block_start}}
+          \s*
```

Comment:
you could put the `\s*` into the `fenced_code_block_start` variable instead to avoid needing to repeat it for each language's code fence.

Reply:
Sure, I'll fix this. I wasn't sure if that would be possible as `{{denced_code_block_start}}` is used in a few other places as well.